### ISSUE TITLE
Load signs

### ISSUE THREAD

**Issue by Hedone** [50k-Customs/DAF-XF#2]
Hello 50Keda,

thx for improve and sharing this truck. I love DAFs.

1) I wish use load signs by Sparex91, maxx2504... and thinking create a personal pack join plates by Volvo 2013 by Ohaha as well (via text editor, it is just I know using)

Could you improve this truck, this way players can use these signs/plates also in your truck? maybe creating another layer more to front of the truck.

http://ets2.lt/en/?s=Warning+Signs+Sparex91
http://ets2.lt/en/trucks-signs/

![2015-08-03_00006](https://cloud.githubusercontent.com/assets/13356154/9051748/6eb7806c-3a35-11e5-8d06-942e5b11bc3f.jpg)
![2015-08-03_00007](https://cloud.githubusercontent.com/assets/13356154/9051757/77c4b08a-3a35-11e5-8394-75340e6ab8fa.jpg)

2) Could you adjust to vertical two dots?  2015-08-03_00007.jpg

thx since now (sorry poor english. I hope you understand me)

**Comment by simon50keda** [comment]
Hi Hedone,
1) Please show me where this "points" shall be positioned and I will create new roof grill for that.
2) You mean move them a little bit back?

**Comment by Hedone** [comment]
Hi again 50Keda.

After write to you I did try to use sign plates in [akcessoria tuning by MTP.scs] and I realized that MTP made two versions to same plate according Tablica Klasy 7B  and Tablica Klasy 7B version 2 These plates are ok because player can see it entire. Are not back in steel of grill. I do not know how MTP did this.

Plates made by Maxxx2504 and Sparex91 are not ok because player can not see entire. So I think that problem is not the dot red in truck but the way how they made. it is dificult learn how to do this (what software program?)?... different position like 2Tablica Klasy 7B ?
![2015-08-04_00009](https://cloud.githubusercontent.com/assets/13356154/9062348/46a63e20-3a98-11e5-8288-2d57e1e8eeea.jpg)
![2015-08-04_00008](https://cloud.githubusercontent.com/assets/13356154/9062349/46a99624-3a98-11e5-90e7-0ccbae864703.jpg)
![2015-08-04_00007](https://cloud.githubusercontent.com/assets/13356154/9062353/46b6123c-3a98-11e5-9b58-38a0b05fe1d8.jpg)
![2015-08-04_00006](https://cloud.githubusercontent.com/assets/13356154/9062351/46b1d848-3a98-11e5-8892-81809f31db22.jpg)
![2015-08-04_00004](https://cloud.githubusercontent.com/assets/13356154/9062350/46b1ce02-3a98-11e5-9036-fd07c8e3e61e.jpg)
![2015-08-04_00003](https://cloud.githubusercontent.com/assets/13356154/9062352/46b5c3e0-3a98-11e5-98a1-77f49ed4a854.jpg)

if I can suggest add layer of dot point...

![_2015-07-30_00011](https://cloud.githubusercontent.com/assets/13356154/9062863/33ac158a-3a9b-11e5-82ec-cdd5e42695b4.jpg)

TY

**Comment by simon50keda** [comment]
Yeah, the thing is that some of the plates are moved from zero position, that's why you see them properly ;)

Anyway I will see what I can do.

**Comment by Hedone** [comment]
TYVM for your patience
